openshift/kustomize/overlays/openshift/kustomization.yaml

```diff
@@ -23,3 +23,7 @@ patches:
       name: controller-manager
     path: patches/manager_deployment_mount_etc_containers.yaml
   - path: patches/manager_namespace_privileged.yaml
+  - target:
+      kind: Deployment
+      name: controller-manager
+    path: patches/manager_deployment_mount_auth_host.yaml
```

Comment:
Nit: group this right after the mount_etc_containers stanza (before the namespace_privileged one)